### PR TITLE
Move share-permissions property namespace

### LINKED ISSUE #23741: Namespace of share-permission webdav property should not be the ownCloud namespace
With https://github.com/owncloud/core/pull/22834 just in we add a new webdav property `oc:share-permissions`.

However, @LukasReschke pointed out that we may want to use a more generic namespace to make interaction with other projects easier. We then also need to document in that namepscae what the permissions means.

FYI @karlitschek

### PR DESCRIPTION
**Pull request by rullzer**
Fixes https://github.com/owncloud/core/issues/23741

@DeepDiver1975 as discussed

CC: @nickvergessen @MorrisJobke @LukasReschke @PVince81

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Integration tests fail.

**Comment by rullzer** [comment]
Aaah tests updated...

**Comment by schiesbn** [comment]
makes sense.... :+1: if tests passed

**Comment by LukasReschke** [comment]
:+1:

File: apps/dav/lib/connector/sabre/filesplugin.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ class FilesPlugin extends ServerPlugin {
 	const FILEID_PROPERTYNAME = '{http://owncloud.org/ns}id';
 	const INTERNAL_FILEID_PROPERTYNAME = '{http://owncloud.org/ns}fileid';
 	const PERMISSIONS_PROPERTYNAME = '{http://owncloud.org/ns}permissions';
-	const SHARE_PERMISSIONS_PROPERTYNAME = '{http://owncloud.org/ns}share-permissions';
```
Comment (DeepDiver1975) [inline apps/dav/lib/connector/sabre/filesplugin.php]: Since when is '{http://owncloud.org/ns}share-permissions' being used?

Comment (DeepDiver1975) [inline apps/dav/lib/connector/sabre/filesplugin.php]: we might want to think about supporting both namespaces

Comment (rullzer) [inline apps/dav/lib/connector/sabre/filesplugin.php]: It was merged 2 weeks ago (so only master).

Comment (DeepDiver1975) [inline apps/dav/lib/connector/sabre/filesplugin.php]: grat - thx

**Pull request merged**